spec/features/drivers/user_can_create_new_driver_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'user can create new driver' do
+  background do
+    visit(drivers_path)
+    click_link('New driver')
+  end
+
+  scenario 'user can fill in new driver spec' do
+    fill_in('Name', with: 'John')
+    fill_in('License number', with: 'licensenumber')
+    fill_in('Mobile number', with: '87654321')
+    fill_in('Email address', with: 'email@email.com')
+    fill_in('Age', with: '18')
+    fill_in('Notes', with: 'notes')
+    choose('driver[status]', option: 'true')
+    click_button('Create Driver')
+
+    expect(page).to have_content('Drivers Index')
+    expect(current_path).to eq(drivers_path)
+
+    within('tbody') do
+      within(all('tr')[0]) do
+        expect(page).to have_content('John')
+        expect(page).to have_content('licensenumber')
+        expect(page).to have_content('87654321')
+        expect(page).to have_content('email@email.com')
+        expect(page).to have_content('18')
+        expect(page).to have_content('notes')
+        expect(page).to have_content('true')
+      end
```

Comment:
Good assertions! we can go one step further to make sure these values are indeed coming from the db, from the record we just created. It's worth adding
```
newly_created_driver = Driver.order(:created_at).last
expect(newly_created_driver.name).to eq('John')
# the same for the other attributes
```
it may seem like overkill at first but it will catch bugs for you in the long run